fabric-api-base/src/testmodClient/java/net/fabricmc/fabric/test/base/client/FabricClientTestHelper.java

```diff
@@ -200,7 +200,7 @@ private static void waitFor(String what, Predicate<MinecraftClient> predicate, D
 				throw new RuntimeException("Timed out waiting for " + what);
 			}
 
-			waitFor(Duration.ofSeconds(1));
+			waitFor(Duration.ofMillis(10));
```

Comment:
I'd increase this a bit, 50ms might be better.

Reply:
Sure thing! I've changed it to 50ms.